Code/MinimalLib/common.h

```diff
@@ -913,8 +913,13 @@ std::string generate_aligned_coords(ROMol &mol, const ROMol &templateMol,
       std::for_each(match.begin(), match.end(),
                     [](auto &pair) { std::swap(pair.first, pair.second); });
       MolTransforms::transformConformer(mol.getConformer(), trans);
+      auto zRot = trans.getVal(2, 2);
+      if (zRot < 0.0 && fabs(zRot) > 0.99) {
```

Comment:
This is likely a stupid question, but why isn't this just:
```suggestion
      if (zRot < -0.99 ) {
```

Reply:
I'd love to have a brilliant reason for this, but I can't find one at the moment, other than low glycemia when I wrote that.